### ISSUE TITLE
OOX-22/FE Distress Beacon with fix

### ISSUE THREAD

**Issue by scotty0100** [classicdb/database#424]
The random drop distress beacon doesn't seem to drop from random creatures in Feralas

The drop rate is currently -100, so as far as I understand that drops only if you are on the quest that requires it..and the only quest that requires this item is the one you actually receive from this book, making it impossible for it to drop.

Following SQL will make it a random drop chance about 1% drop rate. (Used Magelo & WoWHead for rough drop rates). Also changed NPC walk speed of chicken to match the other two Distress Beacon quests. This one walks extremely slow in comparison.

```
UPDATE `creature_template` SET `SpeedWalk`='2', `SpeedRun`='0.714286' WHERE (`Entry`='7807');
INSERT INTO `reference_loot_template` (`entry`, `item`, `ChanceOrQuestChance`, `groupid`) VALUES ('8705', '8705', '0', '1');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5229') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5232') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5234') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5236') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5237') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5238') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5239') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5240') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5241') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5244') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5245') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5246') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5247') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5249') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5251') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5253') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5254') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5255') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5258') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5260') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5262') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5268') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5272') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5274') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5276') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5278') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5286') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5287') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5288') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5292') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5293') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5295') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5296') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5297') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5299') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5300') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5304') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5305') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5306') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5307') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5308') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5327') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5328') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5331') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5332') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5333') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5334') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5335') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5336') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5337') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5343') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5346') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5347') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5349') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5350') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5352') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5356') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5357') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5358') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5359') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5360') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5361') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5362') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5363') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5364') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5366') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5461') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5462') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='5881') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='7584') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='7725') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='7726') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='7727') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='7848') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='8075') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='8136') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='14603') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='14604') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='14638') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='14639') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='14640') AND (`item`='8705');
UPDATE `creature_loot_template` SET `ChanceOrQuestChance`='10', `groupid`='1', `mincountOrRef`='-8705' WHERE (`entry`='14661') AND (`item`='8705');
```

Tested to see if dropped after these changes, and it dropped about after 50 or so creatures killed so seems to be dropping.

Also tested the escort chicken quest this book leads to and works ok :+1:

**Comment by cala** [comment]
Thank you for this report and the proposed fixed.

You are right, if the drop chance is negative, the player must have the quest related to the item to have it dropping. In this case, the quest was unobtainable. However, you don't need to use a reference template for the item and so many queries. Assuming the drop rate is 2% (as it is already set for some entries), the following query is enough.

``` SQL
UPDATE `creature_loot_template` SET  `ChanceOrQuestChance` = 2 WHERE `item` = 8705;
```

Regarding the speed update, I understand you probably took the values from another DB entries (or another DB). However, I'm puzzled by the fact that walkspeed is greater than run speed.

**Comment by scotty0100** [comment]
Oh OK, first time I've meddled with loot tables and was using another similar quest drop item for reference. Foror's Compendium book in Dire Maul.

And yeah I did find that strange as well, also noticed all the HPs of the chicken didn't match across all 3 zones. But I suppose that will change with the update your about to do anyway, right?

Perhaps we should just set run/walk speed for all the chickens to 2? That felt about right when I was doing the quests. No need for the run speed to be slower. I checked a few YouTube videos and it seemed to be about right.
